service/src/main/kotlin/no/nav/su/se/bakover/service/vedtak/VedtakServiceImpl.kt

```diff
@@ -43,4 +46,25 @@ class VedtakServiceImpl(
 
         return GjeldendeVedtaksdata(periode, vedtakSomKanRevurderes, clock).right()
     }
+
+    /*
+    * Syftet er å vise ett "snapshot" av grunnlagsdata vid tidspunktet for ett tidligere vedtak.
+    * */
+    override fun hentTidligereGrunnlagsdataForVedtak(
```

Comment:
Mulig å jobbe litt med navnet på denne? "Tidligere" føles å bære med seg at det eksisterer flere "versjoner" av grunnlagsdataene for dette vedtaket. Kanskje noe i duren av "GrunnlagFørIverksettelseAvVedtak", "HistoriskGrunnlagForVedtaksperiode"  eller lignende kunne fungert?

Kan kanskje diskuteres om denne hører hjemme på `GrunnlagService` eller `VedtakService`, kunne kanskje sett for oss at vi hadde en generell funksjon på grunnlag som kunne gitt oss "historiske" grunnlagsdata, f.eks ved å sende inn en periode og en "max-vedtaks-opprettelse" (så kunne denne funksjonen kalt `GrunnlagService.grunnlagsdataHistorikk(vedtak.periode, vedtak.opprettet)` ellerno.

Reply:
Den funksjonen er väldigt lik det som allerede eksister i den filen där, så er naturligt att de hänger på samma sted. Sen en snabb blick i `GrunnlagService` så ser det ut som det er kun funktionalitet knyttet til henting och lagring av enskilde grunnlag bara.

Tänker att det er greit å ha den där den är, men kan bytte navn iallafall!